assets/css/build/style-style.scss.css

```
@@ -1 +1,869 @@
+  margin-bottom: 20px;
+  padding-top: 0;
+}
+.iawmlf_settings_postbox h2 {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid #c3c4c7;
+  padding: 10px;
+  margin: 0;
+  font-size: 14px;
+  padding: 8px 12px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)